### PR TITLE
Design improve

### PR DESCRIPTION
**Pull request by ReynardEra**
Fix design issues where UI looks different on mobile and pc

### CODE REVIEW

**Comment by ReynardEra** [comment]
Wrong branch

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-
-import AccountScreen from './accountScreen/accountScreen';
-import Login from './Login/Login_index';
+import AccountScreen from './components/accountScreen/accountScreen';
+import Login from './components/login/loginIndex';
 class App extends Component {
   constructor(props){
-    super(props)
+    super(props);
+    this.props = props;
```
Comment (KimberleighBeyl) [inline src/App.js]: remove

File: src/components/accountScreen/accountScreen.js [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+        props.getUserChats(this.props.allMsgs);
+        props.getMsgs(props.currentUser.userID, props.receiver.userID);
+
+        this.filterList = this.filterList.bind(this);
+        //this.state = { initialItems: initialItems, items: initialItems };
+
+    }
+
+    componentWillReceiveProps(newProps) {
```
Comment (KimberleighBeyl) [inline src/components/accountScreen/accountScreen.js]: remove

File: src/App.js (hunk 1 above)
Comment (ReynardEra) [reply src/App.js]: Removed

File: src/components/headerNav/headerNav.js [hunk 3]
```diff
@@ -4,24 +4,22 @@ import { bindActionCreators } from 'redux';
...
-import { addMsg, getMsgs, getChats } from '../reducers/Chat';
+import { addMsg, getMsgs, getChats } from '../../reducers/Chat';
 
 
 class HeaderNav extends Component {
 
     constructor(props){
       super(props);
-    }
+      this.props = props;
```
Comment (KimberleighBeyl) [inline src/components/headerNav/headerNav.js]: remove

File: src/components/accountScreen/accountScreen.js (hunk 2 above)
Comment (ReynardEra) [reply src/components/accountScreen/accountScreen.js]: Removed

File: src/components/registration/stepOne.js [hunk 4]
```diff
@@ -1,14 +1,34 @@
...
+import {
+  addMsg,
+  getMsgs,
+  getChats
+} from '../../reducers/Chat'
+
 class StepOne extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
```
Comment (KimberleighBeyl) [inline src/components/registration/stepOne.js]: remove

File: src/components/registration/stepTwo.js [hunk 5]
```diff
@@ -1,10 +1,32 @@
...
+import {
+  addMsg,
+  getMsgs,
+  getChats
+} from '../../reducers/Chat'
+
 class StepTwo extends React.Component {
+    constructor(props) {
+        super(props);
+        this.props = props;
```
Comment (KimberleighBeyl) [inline src/components/registration/stepTwo.js]: remove

File: src/components/headerNav/headerNav.js (hunk 3 above)
Comment (ReynardEra) [reply src/components/headerNav/headerNav.js]: Removed

File: src/components/registration/stepOne.js (hunk 4 above)
Comment (ReynardEra) [reply src/components/registration/stepOne.js]: removed

File: src/components/registration/stepTwo.js (hunk 5 above)
Comment (ReynardEra) [reply src/components/registration/stepTwo.js]: removed

**Pull request merged**